src/pages/about/whoWeAre.md

```diff
@@ -0,0 +1,24 @@
+---
+templateKey: 'about-page'
+path: /whoWeAre
+title: Who We are
+---
+### [Stephen Drollinger](https://github.com/)
+
+
+
+### [John Favorite](https://github.com/OldCrowEW)
+
+
+
+### [Jonathan Meyers](https://github.com/jmymy)
+
+Jonathan Meyers is the Head of Infrastructure at Cybrary where he is responsible for designing, maintaining, and securing all corporate infrastructure including their Security Enablement platform supporting over 200 companies and 2.5 million users worldwide. He previously worked as a Senior DevOps and Senior Operations Engineer at Forcepoint (formally RedOwl Analytics prior to their acquisition) where he oversaw the operations and deployment of their hosted and on-prem UEBA/E-Surveillance product. Jonathan holds an Information Technology degree from The United States Military Academy at West Point.
+
+### [Wayne Pascoe](https://github.com/WTPascoe)
+
```

Comment:
Wayne Pascoe is the head of DevOps for an external customer facing product team at J.P. Morgan responsible for modernising the platform and processes to deliver a SaaS product. He is also responsible for developing the support operating model for a number of institutional clients external to the company.

Before this, Wayne worked as a Senior Engineer and Head of DevOps at Beacon Platform and was responsible for client deployments and support in both cloud and on-premises environments. He was also a key part of the team that implemented processes, controls, and reporting to achieve SOC 1 and SOC 2 attestation.

Wayne brings a wealth of knowledge and experience from working in some of the world’s largest banks and exciting start-ups.